src/cascadia/TerminalSettingsModel/Resources/en-US/Resources.resw

```diff
@@ -740,4 +740,274 @@
+  <data name="RenameWindow" xml:space="preserve">
+    <value>Rename Window</value>
+  </data>
+  <data name="OpenWindowRenamer" xml:space="preserve">
+    <value>Open Window Renamer</value>
+  </data>
+  <data name="DisplayWorkingDirectory" xml:space="preserve">
+    <value>Display Working Directory</value>
+  </data>
+  <data name="SearchForText" xml:space="preserve">
+    <value>Search For Text</value>
+  </data>
+  <data name="GlobalSummon" xml:space="preserve">
+    <value>Global Summon</value>
+  </data>
+  <data name="QuakeMode" xml:space="preserve">
+    <value>Quake Mode</value>
+  </data>
+  <data name="FocusPane" xml:space="preserve">
+    <value>Focus Pane</value>
+  </data>
+  <data name="OpenSystemMenu" xml:space="preserve">
+    <value>Open System Menu</value>
+  </data>
+  <data name="ExportBuffer" xml:space="preserve">
+    <value>Export Buffer</value>
+  </data>
+  <data name="ClearBuffer" xml:space="preserve">
+    <value>Clear Buffer</value>
+  </data>
+  <data name="MultipleActions" xml:space="preserve">
+    <value>Multiple Actions</value>
+  </data>
+  <data name="Quit" xml:space="preserve">
+    <value>Quit</value>
+  </data>
+  <data name="AdjustOpacity" xml:space="preserve">
+    <value>Adjust Opacity</value>
+  </data>
+  <data name="RestoreLastClosed" xml:space="preserve">
+    <value>Restore Last Closed</value>
+  </data>
+  <data name="SelectAll" xml:space="preserve">
+    <value>Select All</value>
+  </data>
+  <data name="SelectCommand" xml:space="preserve">
+    <value>Select Command</value>
+  </data>
+  <data name="SelectOutput" xml:space="preserve">
+    <value>Select Output</value>
+  </data>
+  <data name="MarkMode" xml:space="preserve">
+    <value>Mark Mode</value>
+  </data>
+  <data name="ToggleBlockSelection" xml:space="preserve">
+    <value>Toggle Block Selection</value>
+  </data>
+  <data name="SwitchSelectionEndpoint" xml:space="preserve">
+    <value>Switch Selection Endpoint</value>
+  </data>
+  <data name="Suggestions" xml:space="preserve">
+    <value>Suggestions</value>
+  </data>
+  <data name="ColorSelection" xml:space="preserve">
+    <value>Color Selection</value>
+  </data>
+  <data name="ShowContextMenu" xml:space="preserve">
+    <value>Show Context Menu</value>
+  </data>
+  <data name="ExpandSelectionToWord" xml:space="preserve">
+    <value>Expand Selection To Word</value>
+  </data>
+  <data name="CloseOtherPanes" xml:space="preserve">
+    <value>Close Other Panes</value>
+  </data>
+  <data name="RestartConnection" xml:space="preserve">
+    <value>Restart Connection</value>
+  </data>
+  <data name="ToggleBroadcastInput" xml:space="preserve">
+    <value>Toggle Broadcast Input</value>
+  </data>
+  <data name="OpenScratchpad" xml:space="preserve">
+    <value>Open Scratchpad</value>
+  </data>
+  <data name="OpenAbout" xml:space="preserve">
+    <value>Open About</value>
+  </data>
+  <data name="QuickFix" xml:space="preserve">
+    <value>Quick Fix</value>
+  </data>

```

Comment:
So, the type here is "Quick Fix", but there's no action args and the generated name is "Open quick fix menu". Should we just use the generated name here?

Generalize this comment to all the other actions that have no args. I think the list can basically be found in ActionAndArgs.cpp by looking at `GenerateName()`